moderation/commands.go

```diff
@@ -1033,6 +1047,24 @@ var ModerationCommands = []*commands.YAGCommand{
 				return fmt.Sprintf("Could not find warning with ID `%d`", warningID), nil
 			}
 
+			if config.UnwarnSendToModlog && config.ActionChannel != 0 {
+				userid, err := strconv.ParseInt(warning.UserID, 10, 64)
+				if err != nil {
+					return "Failed parsing user ID", err
+				}
+				user := bot.GetUsers(parsed.GuildData.GS.ID, userid)[0]
+
+				message := parsed.Args[1].Str()
+				if config.UnwarnIncludeWarnReason {
+					message = fmt.Sprintf("%s\n~~%s~~", message, warning.Message)
```

Comment:
I wonder if this might be more clear if the previous, strikenthrouwgh reason was first, followed by the new one. What are your thoughts?

Reply:
I tested that way, but having the warn comment first is weird, because the action is the unwarn